multinet/api.py

```diff
@@ -326,3 +328,37 @@ def delete_table(workspace: str, table: str) -> Any:
     """Delete a table."""
     db.delete_table(workspace, table)
     return table
+
+
+@bp.route("/uploads", methods=["POST"])
+def create_upload() -> str:
+    """Create a collection for multipart upload."""
+
+    sysdb = db.db("_system")
+
+    # note: arangodb collections can't start with a number
+    upload_id = f"upload_{uuid4().hex}"
+
+    sysdb.create_collection(upload_id)
+    return upload_id
+
+
+@bp.route("/uploads/<upload_id>/chunk", methods=["POST"])
+@use_kwargs({"sequence": fields.Str()})
+def chunk_upload(upload_id: str, sequence: str) -> Any:
+    """Upload a chunk to the specified collection."""
+    chunk = dict(request.files)["chunk"].read()
```

Comment:
What if there are no files associated with the request?

Reply:
Also, do you know if there's a way to read out the file size first, and abort with an error if it's above a certain limit? Otherwise this could be used to DoS the server.

Reply:
> Also, do you know if there's a way to read out the file size first, and abort with an error if it's above a certain limit? Otherwise this could be used to DoS the server.

Yes, looks like Flask supports this.
https://flask.palletsprojects.com/en/1.1.x/config/#MAX_CONTENT_LENGTH

From what I understand, Flask checks the actual file length and not just the `CONTENT_LENGTH` header of the request, so this should cover us in regards to DoS attacks